src/lib/xively/blueprint.ts

```diff
@@ -401,7 +401,7 @@ class Organizations {
       url: this.urlBP
       + 'organizations',
       method: 'POST',
-      body: org,
+      body: JSON.stringify(org),
```

Comment:
This doesn't seem right to me, in other projects we do not stringify for this api call

Reply:
Here we are using `fetch` and it was throwing an error if a string isn't provided.